state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -849,6 +852,19 @@ pub mod pallet {
 									swap.to
 								);
 							},
+						SwapType::TransactionFee => {
```

Comment:
Should we put the enum with the extra data like the swap id? it would be repeated information. Or should I make a new enum with the data stripped away?

Reply:
Let's live with the duplication for now - we can address this with PRO-1282.